### PR TITLE
Extracting query times

### PR DESCRIPTION
**Pull request by elshize**

### CODE REVIEW

**Comment by elshize** [comment]
In progress. Still want to: (a) add an option to suppress logging (`--silent`? any other suggestion?), and (b) write logs to stderr by default (now it's written to stdout).

**Comment by elshize** [comment]
@amallia Ready for review.

File: src/queries.cpp [hunk 1]
```diff
@@ -216,45 +255,55 @@ int main(int argc, const char **argv) {
     auto *terms_opt =
         app.add_option("--terms", terms_file, "Text file with terms in separate lines");
     app.add_flag("--nostem", nostem, "Do not stem terms")->needs(terms_opt);
+    app.add_flag("--extract", extract, "Extract individual query times")->needs(terms_opt);
```
Comment (amallia) [inline src/queries.cpp]: I dont think `extract` needs `terms_opt `

**Review by amallia — CHANGES_REQUESTED** [review]

File: docs/source/query_index.md [hunk 2]
```diff
@@ -2,25 +2,31 @@
...
+                                  Query algorithm
+      -i,--index TEXT REQUIRED    Collection basename
+      -w,--wand TEXT              Wand data filename
+      -q,--query TEXT             Queries filename
+      --compressed-wand           Compressed wand input file
+      -k UINT                     k value
+      -T,--thresholds TEXT        k value
+      --terms TEXT                Text file with terms in separate lines
+      --nostem Needs: --terms     Do not stem terms
+      --extract Needs: --terms    Extract individual query times
```
Comment (amallia) [inline docs/source/query_index.md]: Same here

**Review by amallia — CHANGES_REQUESTED** [review]

**Comment by amallia** [comment]
Everything looks good, a part a small think related to CLI.
After fixing this we can merge it!

**Review by amallia — APPROVED** [review]

**Pull request merged**